superduper/misc/download.py

```diff
@@ -500,3 +510,17 @@ def predict_batches(self, dataset):
         if self.postprocess:
             results = [self.postprocess(r) for r in results]
         return results
+
+
+def hash_uri(uri: str) -> str:
```

Comment:
Please move this to `superduper.misc.hash`

Reply:
done